static/js/popovers.js

```diff
@@ -347,6 +348,27 @@ exports.show_user_profile = function (user) {
     );
 };
 
+exports.show_user_info_popover = function (element, user) {
+    const last_popover_elem = current_user_info_popover_elem;
+    exports.hide_all();
+    if (last_popover_elem !== undefined && last_popover_elem.get()[0] === element) {
+        return;
+    }
+    const elt = $(element);
+    if (elt.data("popover") === undefined) {
```

Comment:
Is there a situation where we expect this to ever be false?

Reply:
No, there is no situation for this to be false, I have removed this condition.